pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -116,6 +116,24 @@ func (r *ReconcileMasterUserRecord) Reconcile(request reconcile.Request) (reconc
 			logger.Error(err, "unable to add finalizer to MasterUserRecord")
 			return reconcile.Result{}, err
 		}
+		// migration for CRT-1075: add an annotation with the email address (same as for associated UserSignup resource)
```

Comment:
Don't forget to add a task (sub-task in JIRA?) to clean up this migration step when done.

Reply:
already done: https://issues.redhat.com/browse/CRT-1074 😜

Reply:
It's not done until it's in the PR description ;)